src/data/pokemon/base_stats.h

```diff
@@ -21213,7 +23276,11 @@ const struct BaseStats gBaseStats[] =
         .growthRate = GROWTH_SLOW,
         .eggGroup1 = EGG_GROUP_AMORPHOUS,
         .eggGroup2 = EGG_GROUP_AMORPHOUS,
-        .abilities = {ABILITY_PIXILATE, ABILITY_PIXILATE},
+        #ifdef BATTLE_ENGINE
+            .abilities = {ABILITY_PIXILATE, ABILITY_PIXILATE},
```

Comment:
All the Megas seem to have duplicate abilities. This should be looked into to see why and if it's necessary.